zeppelin-web/src/app/notebook/notebook.html

```diff
@@ -14,6 +14,29 @@
 <!-- Here the controller <NotebookCtrl> is not needed because explicitly set in the app.js (route) -->
 <div ng-include src="'app/notebook/notebook-actionBar.html'"></div>
 <div style="padding-top: 36px;">
+  <!-- permissions -->
+  <div ng-if="showPermissions" class="permissions">
+    <div>
+      <h4>Note Permissions (Only note owners can change)</h4>
+    </div>
+    <hr />
+    <div>
+      <p>
+        Enter comma separated users and groups in the fields. Empty set means anyone can do the operation.
+      </p>
+      <div class="notePermissions"
+           data-ng-model="permissions">
+        <p>Owners : <input ng-list ng-model="permissions.owners"> Owners can change permissions, read and write the note. </p>
+        <p>Readers : <input ng-list ng-model="permissions.readers"> Readers can only read the note.</p>
+        <p>Writers : <input ng-list ng-model="permissions.writers"> Writers can read and write the note.</p>
```

Comment:
When each 'Owners', 'Readers', 'Writers' is empty, default behavior is allowing all.
But some people may expect default behavior is deny all. So how about make user quickly catch the default behavior?

Adding whildcard(*) at placeholder or anyother way.

`<input ng-model ..... placeholder="*"> ....`